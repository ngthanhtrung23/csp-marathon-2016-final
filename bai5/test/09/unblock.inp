1222..
13334.
.5004.
.5664.
.5.877
9998..
