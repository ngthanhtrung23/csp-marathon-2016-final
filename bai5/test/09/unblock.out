17
1 D 2
2 L 1
3 L 1
4 U 1
6 R 2
8 U 1
9 R 2
5 D 1
0 L 1
8 U 3
0 R 1
5 U 1
6 L 2
7 L 2
9 L 1
4 D 3
0 R 2

